### ISSUE TITLE
Feature: named arenas

### ISSUE THREAD

**Issue by mooshbeef** [Pocketkid2/Survival-Games#7]
The index/identifier system is faily basic and is not easily rememberable.
Most addons allow you to give a name to a region as its identifier.

Probably best to first add names besides the current ID system, but ensure they are unique.
Then later you can (if you want) completely replace them.

**Comment by Pocketkid2** [comment]
Yes yes yes yes yes yes yes yes I do want to fix that it'll come in the future
